devel/smake/Portfile

```diff
@@ -1,41 +1,71 @@
-long_description	Smake is a highly portable make program with automake \
-				features. It is currently mainly targeted to be used \
-				with the makefiles system. 
+PortSystem              1.0
+PortGroup               makefile 1.0
+PortGroup               compiler_blacklist_versions 1.0
 
-homepage		https://sourceforge.net/projects/s-make/
-master_sites	sourceforge:project/s-make
-use_bzip2		yes
+name                    smake
+version                 1.6
+revision                0
+categories              devel
+platforms               darwin
+maintainers             {hotmail.com:amtor @RobK88} \
+                        openmaintainer
+license                 CDDL
+description             highly portable make program with automake features
+long_description        Smake is a highly portable make program with automake \
+                        features. It is currently mainly targeted to be used \
+                        with the Schily SING makefile system.
 
-checksums           rmd160  f683a4051273b3e24e45a2b435ddb23509db55bf \
-                    sha256  27566aa731a400c791cd95361cc755288b44ff659fa879933d4ea35d052259d4
+homepage                https://sourceforge.net/projects/s-make/
+master_sites            sourceforge:schilytools
 
-use_configure	no
+checksums               sha1    876b30c2f63e1ba7d682d0cf58789d26a527f070 \
+                        rmd160  9ad475b67122f487844162ea2e6e3659ea25579f \
+                        sha256  7c93f43d22c6e3de0eafe018ceb26ddd5cc6c543dc301f307d1e6ca313248aa4 \
+                        size    4952788
+
+# If the version number in ${distname} changes, please remember to also update ${livecheck.version}
+distname                schily-2021-09-18
+
+use_bzip2               yes
+
+post-extract {
+                        copy ${filespath}/Gmake.smake ${worksrcpath}
+}
+
+patchfiles              001-Makefile-Patch.diff
 
 # https://trac.macports.org/ticket/31616
-compiler.blacklist-append llvm-gcc-4.2 {clang < 318}
-
-build.type		gnu
-build.args      CC=${configure.cc}
-use_parallel_build  no
-
-destroot.args       CC=${configure.cc}
-destroot.destdir	INS_BASE=${destroot}${prefix} \
-					MANDIR=man \
-					INSUSR=`id -u` INSGRP=`id -g`
-post-destroot	{
-	file delete -force ${destroot}${prefix}/share/man/man5 \
-		${destroot}${prefix}/lib ${destroot}${prefix}/include
+compiler.blacklist      llvm-gcc-4.2 macports-llvm-gcc-4.2 {clang < 300}
+
+configure.ldflags-append -lintl
+
+depends_build-append    port:gmake \
+                        port:gettext-runtime
+                        
+depends_lib-append      port:gettext-runtime
+
+build.type              gnu
+                        
+build.args-append       INS_BASE="${destroot}${prefix}" \
+                        INS_RBASE="${destroot}${prefix}" \
+                        DEFOSINCDIRS="${prefix}/include ./" \
+                        CC_OPT=${configure.optflags} \
+
+use_parallel_build      no
+
+universal_variant       no
```

Comment:
Any reason to disable universal variant?

Reply:
If you look at the ports for any of the "Schily" software, like `star` and `cdrtools`, you will notice that none of the ports support the creation of universal variants.  I suspect that one cannot create universal variants with the Schily SING Makeflie system (using `smake`).